### PR TITLE
Implement obfuscating app settings

### LINKED ISSUE #640: Obfuscate environment variable values
Default should be to not show the value in the tree nodes or tooltips and still allow editing. Add a config option to override this behavior.

### PR DESCRIPTION
**Pull request by nturinski**
Fixes #640 
Relies on this PR https://github.com/Microsoft/vscode-azuretools/pull/353

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -274,6 +274,11 @@
                 "command": "appService.RevealConnectionInAppSettings",
                 "title": "Reveal in Application Settings",
                 "category": "Azure App Service"
+            },
+            {
+                "command": "appService.toggleValueVisability",
```
Comment (EricJizbaMSFT) [inline package.json]: Visability -> Visibility

File: package.json [hunk 2]
```diff
@@ -274,6 +274,11 @@
...
+                "title": "Reveal App Settings",
```
Comment (EricJizbaMSFT) [inline package.json]: Remove or change the title to reflect the actual command

File: src/commands/toggleValueVisability.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { AppSettingTreeItem } from "vscode-azureappservice";
+
+export async function toggleValueVisability(node: AppSettingTreeItem): Promise<void> {
+    node.toggleValueVisability();
+    await node.refresh();
```
Comment (EricJizbaMSFT) [inline src/commands/toggleValueVisability.ts]: Shouldn't `toggleValueVisability` handle the refresh?

Comment (EricJizbaMSFT) [reply src/commands/toggleValueVisability.ts]: Also if this ends up being one line of code, I think I'm fine with it going directly in extension.ts

Comment (nturinski) [reply src/commands/toggleValueVisability.ts]: Good point

File: package.json (hunk 1 above)
Comment (nturinski) [reply package.json]: 😊

**Comment by nturinski** [comment]
image
image

**Pull request merged**